src/lit-html.ts

```diff
@@ -57,26 +57,28 @@ export const svg = (strings: TemplateStringsArray, ...values: any[]) =>
  *     html`<input type="checkbox" ?checked=${value}>`
  */
 export const partCallback =
-    (instance: TemplateInstance,
-     templatePart: TemplatePart,
-     node: Node): Part => {
-      if (templatePart.type === 'attribute') {
-        const name = templatePart.name!;
-        const prefix = name[0];
-        if (prefix === '.') {
-          return new PropertyPart(
-              instance, node as Element, name.slice(1), templatePart.strings!);
-        }
-        if (prefix === '@') {
-          return new EventPart(instance, node as Element, name.slice(1));
-        }
-        if (prefix === '?') {
-          return new BooleanAttributePart(
-              instance, node as Element, name.slice(1), templatePart.strings!);
-        }
-      }
-      return defaultPartCallback(instance, templatePart, node);
-    };
+    (instance: TemplateInstance, templatePart: TemplatePart, node: Node):
+        Part[] => {
+          if (templatePart.type === 'attribute') {
+            const name = templatePart.name!;
+            const prefix = name[0];
+            if (prefix === '.') {
+              const comitter = new PropertyCommitter(
+                  node as Element,
+                  templatePart.name!.slice(1),
+                  templatePart.strings!);
+              return comitter.parts;
```

Comment:
It seems somewhat weird that you make `AttributePart`s for properties, but I'm not sure this is worth addressing, wdyt?

Reply:
I actually do this we should add a trivial PropertyPart subclass of AttributePart just in case there's a need for a difference in behavior in the future and we want that change to be less likely to break anyone.

I'll add this.

Reply:
This is done btw